docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
Still need specific channel names so no channel becomes a firehose. Impossible to predict that to some extent. Imagine likes on hot content on a big network...